packages/round-manager/src/features/api/programService.ts

```diff
@@ -0,0 +1,124 @@
+// import { ethers } from "ethers"
+import { api } from "."
+import { global } from "../../global"
+
+
+export interface Program {
```

Comment:
thoughts on having the interace in it's own file to make the service cleaner / smaller ?

Reply:
done ✅